server.js

```diff
@@ -0,0 +1,39 @@
+const express = require('express');
+const db = require('./data/db')
+
+const port = 5555;
+const server = express();
+server.use(express.json());
+
+server.get('/', (req, res) => {
+    // 1st arg route where our reasour can be commnnicated with
+    // 2nd arg callback to deal with sending responses
+    res.send('Hello from ExP');
+});
+
+server.post('/api/users', (req, res) => {
+    // console.log(req.body);
+    if (req.body.name === undefined || req.body.bio === undefined) {
```

Comment:
Good and very specific check for name and bio in here. Consider de-structuring those variables like this: 
```const { name, bio } = req.body; // This "extracts" req.body.name and req.body.bio into name and bio```